ivy/functional/frontends/tensorflow/linalg.py

```diff
@@ -39,3 +39,7 @@ def pinv(a, rcond=None, validate_args=False, name=None):
 
 
 pinv.unsupported_dtypes = ("float16", "bfloat16")
+
+def tensordot(x,y,axes,name=None):
+    return ivy.tensordot(x,y,axes)
+
```

Comment:
You have to add the supported or unsupported dtypes here according to the documentation https://www.tensorflow.org/api_docs/python/tf/tensordot